specs/interop/supervisor.md

```diff
@@ -154,6 +154,13 @@ Describes a block.
 
 #### `DerivedIDPair`
 
+A pair of block identifiers, linking an L2 block (`derived`) to the L1 block (`source`) from which it was derived.
```

Comment:
Sorry for the nit-picking, but could we adjust the description too?

```suggestion
A pair of block identifiers, linking an `derived` block (typically an L2 block) to the `source` (typically an L1 block) from which it was derived.
```

There is some abstract future where we could accept log messages even from the `L1`, and in that case we would probably model a `DerivedIDPair` as `L1:L1` (identical on both sides), because the L1 is derived from itself, so to speak.

Feel free to push back if you think I'm introducing too much nuance here.

Reply:
Ahh. Sorry, I missed it